docs/reference/openapi-rules.md

```diff
@@ -1,3 +1,546 @@
+
+**Recommended:** Yes
+
+### contact-properties
+
+The [info-contact](#info-contact) rule will ask you to put in a contact object, and this rule will make sure it's full of the most useful properties: `name`, `url` and `email`. 
+
+Putting in the name of the developer/team/department/company responsible for the API, along with the support email and help-desk/GitHub Issues/whatever URL means people know where to go for help. This can mean more money in the bank, instead of developers just wandering off or complaining online.
+
+**Recommended:** No
+
+**Good Example**
+
+```yaml
+openapi: "3.0.2"
+info:
+  title: Awesome API
+  description: A very well defined API
+  version: "1.0"
+  contact:
+    name: A-Team
+    email: a-team@goarmy.com
+    url: goarmy.com/apis/support
+```
+
+### example-value-or-externalValue
+
+Examples for `requestBody` or response examples can have an `externalValue` or a `value`, but they cannot have both.
+
+**Recommended:** No
+
+**Bad Example**
+
+```yaml
+paths:
+  /pet:
+    put:
+      operationId: "relace-pet"
+      requestBody:
+        content:
+          'application/json':
+            examples: 
+              foo:
+                summary: A foo example
+                value: {"foo": "bar"}
+                externalValue: 'http://example.org/foo.json' 
+                # marp! no, can only have one or the other
+```
+
+### info-contact
+
+Info object should contain `contact` object. 
+
+Hopefully your API description document is so good that nobody ever needs to contact you with questions, but that is rarely the case. The contact object has a few different options for contact details.
+
+**Recommended:** Yes
+
+**Good Example**
+
+```yaml
+openapi: "3.0.2"
+info:
+  title: Awesome API
+  version: "1.0"
+  contact: 
+    name: A-Team
+    email: a-team@goarmy.com
+```
+
+### info-description
+
+OpenAPI object info `description` must be present and non-empty string.
+
+Examples can contain Markdown so you can really go to town with them, implementing getting started information like where to find authentication keys, and how to use them.
+
+**Recommended:** Yes
+
+**Good Example**
+
+```yaml
+openapi: 3.0.0
+info:
+  version: '1.0.0'
+  title: Descriptive API
+  description: >+
+    Some description about the general point of this API, and why it exists when another similar but different API also exists.
+
+    ## Authentication
+
+    This API uses OAuth2 and tokens can be requested from [Dev Portal: Tokens](https://example.org/developers/tokens).
+```
+
+### info-license
+
+The `info` object should have a `license` key.
+
+It can be hard to pick a license, so if you don't have a lawyer around you can use [TLDRLegal](https://tldrlegal.com/) and [Choose a License](https://choosealicense.com/) to help give you an idea. 
+
+How useful this is in court is not entirely known, but having a license is better than not having a license. 
```

Comment:
:D lol 

> "but having a license is better than not having a license. "

is that a legal advice? ;)

Reply:
It's legal advice as much "a lawyer is better at law than somebody who is not a lawyer" is legal advice. 

I have specifically not given legal advice.